### PR TITLE
[MINOR] Add basic tests and setup

### LINKED ISSUE #34: Test Setup
As the bot is growing quite fast I think it is a good idea to add tests. We need a test setup and if possible a github action to run the tests automatically. I don't think it is necessary to reach 100% test coverage in this step, but we need a starting point.

### PR DESCRIPTION
**Pull request by leonardxfce**
Add jest and split some code to be testeable

Fixes https://github.com/Fubinator/planning-poker-discord-bot/issues/34

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,106 +1,14 @@
+const client = require("./client");
 require("dotenv").config({ path: "../.env" });
 
-const { Client, Collection } = require("discord.js");
```
Comment (leonardxfce) [inline src/index.js:3]: I splited this block of code in order to able to test that code.

File: package.json [hunk 2]
```diff
@@ -7,6 +7,7 @@
   "license": "MIT",
   "private": false,
   "scripts": {
+    "test": "jest --detectOpenHandles --forceExit",
```
Comment (leonardxfce) [inline package.json:10]: `--forceExit` is needed to stop async code in setTimeout code block

**Comment by leonardxfce** [comment]
@Fubinator how it looks for you?

**Comment by Fubinator** [comment]
@leonardxfce it's looking good so far. I'm just trying to figure out how I can merge this to see if anything is broken.

**Review by Fubinator — APPROVED** [review]

**Comment by Fubinator** [comment]
Nice work @leonardxfce. Congratz on your first PR and thank you very much!

**Comment by leonardxfce** [comment]
Many thanks!

**Pull request merged**